src/features/recovery/components/RecoveryEmail/index.tsx

```diff
@@ -53,15 +62,49 @@ const RecoveryEmail = () => {
     )
   }
 
+  const onDelete = async () => {
+    try {
+      await deleteEmailAddress()
+    } catch (e) {
+      const error = asError(e)
+
+      if (isNoContentResponse(error.message)) {
```

Comment:
if nothing is returned from the SDK, it is treated as an error, is that correct?

Reply:
I plan to remove this workaround once we have a fix in the SDK. Currently, the SDK throws for a 204 status because there is a header `content-length: 0` missing. Ideally we should check explicitely for a 204 status there and not throw.